doc/release-notes/5.4-release-notes.md

```diff
@@ -0,0 +1,337 @@
+
+Due to the changes for Conditionally Required Metadata Fields, and a minor update in the citation metadata block to support extra ISO-639-3 language codes, a block upgrade is required. Instructions are provided below.
+
+### Retroactively Store Original File Size
+
+Beginning in Dataverse Software 4.10, the size of the saved original file (for an ingested tabular datafile) was stored in the database. For files added before this change, we provide an API that retrieves and permanently stores the sizes for any already existing saved originals. See [Datafile Integrity API](https://guides.dataverse.org/en/5.4/api/native-api.html#datafile-integrity) for more information.
+
+This was documented as a step in previous release notes, but we are noting it in these release notes to give it more visibility.
+
+### DB Cleanup for Saved Searches
+
+A previous version of the Dataverse Software changed the indexing logic so that when a user links a Dataverse collection, its children are also indexed as linked. This means that the children do not need to be separately linked, and in this version we removed the logic that creates a saved search to create those links when a Dataverse collection is linked.
+
+We recommend cleaning up the db to a) remove these saved searches and b) remove the links for the objects. We can do this via a few queries, which are available in the folder here:
+
+<https://github.com/IQSS/dataverse/raw/develop/scripts/issues/7398/>
+
+There are four sets of queries available, and they should be run in this order:
+
+- ss_for_deletion.txt to identify the Saved Searches to be deleted
+- delete_ss.txt to delete the Saved Searches identified in the previous query
+- dld_for_deletion.txt to identify the linked datasets and Dataverse collections to be deleted
+- delete_dld.txt to delete the linked datasets and Dataverse collections identified in the previous queries
+
+Note: removing these saved searches and links should not affect what users will see as linked due to the aforementioned indexing change. Similarly, not removing these saved searches and links should not affect anything, but is a cleanup of unnecessary rows in the database.
+
+### DB Cleanup for Superusers Releasing without Version Updates
+
+In datasets where a superuser has run the Curate command and the update included a change to the fileaccessrequest flag, those changes would not be reflected appropriately in the published version. This should be a rare occurrence.
+
+Instead of an automated solution, we recommend inspecting the affected datasets and correcting the fileaccessrequest flag as appropriate. You can identify the affected datasets this via a query, which is available in the folder here:
+
+<https://github.com/IQSS/dataverse/raw/develop/scripts/issues/7687/>
+
+### New JVM and DB Options
+
+#### New JVM Options
+
+For installations that were previously running on Dataverse Software 4.x, the following JVM Options have been added in support of Java 11:
```

Comment:
Below we say this:

**(only required for installations previously running Dataverse Software 4.x under Glassfish 4!)**

The same applies here, right? These new JVM options are only new for people who aren't starting with Payara, right? They're new if they are still using the domain.xml from Glassfish 4.1, right?

To me the question is, where did you domain.xml come from?

Reply:
I just came here to ask, can I erase this block - "New JVM optioins" completely? They are already listed in the upgrade instruction below. And I don't think it adds any value to list them here as well. 
It just makes the release notes longer; and they are already huge for the release.

Reply:
@pdurbin - I have removed that entire "new jvm options" section. Again, I feel like that information is *only* relevant in the context of performing the upgrade. So it should be sufficient to explain it in detail the "Upgrade Instructions" section only. 
I did however incorporate your language - that spells out the distinction of having a domain.xml that's inherited-from-glassfish4 vs. one from a Payara distribution - into the relevant paragraph in the upgrade instruction. I agree it makes sense to explain it like that.

Reply:
Removing it here sounds perfect. Thanks, @landreev!

Reply:
P.S. @pdurbin note that a shortened version of that section was put back in the notes in the end; regardless, thanks for the suggestion for the clear "inherited vs. new domain.xml" explanation.